content/yaml-running-builds/starting-builds-automatically.md

```diff
@@ -192,7 +192,7 @@ Environment variables are available under the `env` variable. You can check [bui
 
 Webhook payload is available under the `event` variable. You can check the structure of the webhook payloads that your git provider sends on the **Webhooks** tab in application settings. Note that `event` is not available if the build is started manually from the UI or by a schedule.
 
-For example, this build will continue if the triggering event was *not* a draft pull request update:
+Example 1. This build will continue if the triggering event was *not* a draft pull request update. In other words, it will skip the build if a branch is in a pull request and marked as a draft:
```

Comment:
`In other words, it will skip the build if a branch is in a pull request and marked as a draft:
`
You mean if PR is marked as draft ?